### DISCUSSION TITLE
How can I add relationship between Content and AdminUser?

### DISCUSSION THREAD

**Discussion by darron1217** [strapi/strapi#7822] [Legacy Topics]
I guess it's related to security.
If then, is there a way to workaround it?

**Comment by derrickmehaffy** [comment]
There is not yet, but we do have a PR for it: https://github.com/strapi/strapi/pull/7788

However the relations will be limited only to oneWay and manyWay as to prevent modification of the Admin user settings file.

**Reply by darron1217** [reply]
Great news 👍 

I'll look forward to next release :)

**Reply by darron1217** [reply]
Does editing Admin User settings file causes problem?

**Reply by derrickmehaffy** [reply]
At the moment it's not possible without forking the package as it's not a plugin, you can't extend the `strapi-admin` backend at this time.

We have an open feature request to move the `strapi-admin` package to `strapi-plugin-admin`: https://github.com/strapi/strapi/issues/4838

**Reply by darron1217** [reply]
Thanks. You're so kind :)
